pkg/controller/templateupdaterequest/templateupdaterequest_controller.go

```diff
@@ -208,18 +238,18 @@ func namespaceType(templateRef string) string {
 	return parts[1]
 }
 
-// compareSyncIndexes compares the sync indexes in the given TemplateUpdateRequest status vs the given MasterUserRecord
-// returns `true` if ALL values are DIFFERENT, meaning that all user accounts were updated on the target clusters where the tier is in use
-func (r ReconcileTemplateUpdateRequest) compareSyncIndexes(logger logr.Logger, tur toolchainv1alpha1.TemplateUpdateRequest, mur toolchainv1alpha1.MasterUserRecord) bool {
+// sameSyncIndexes compares the sync indexes in the given TemplateUpdateRequest status vs the given MasterUserRecord
+// returns `false` if ALL values are DIFFERENT, meaning that all user accounts were updated on the target clusters where the tier is in use
```

Comment:
I wonder why we consider the indexes be the same even if just one of them is the same and the rest are changed?

Reply:
I'm not sure I fully understand your question, but the point here is that we want to make sure that all syncIndexes changed, ie, that all user accounts associated with the updated tier have been updated. Does it clarify?

Reply:
Then rename this function to `allSyncIndexesChanged()`? :)
Right now it's confusing. Right now `sameSyncIndexes()` returns true if one index is the same and another one is not. Which is confusing.

Reply:
ok, let me do that

Reply:
done in 011aabc